GlobalConstantsService/data/globalConstants_01.txt

```diff
@@ -21,9 +21,9 @@ int    physicsParams.nominalDAQOnSpillBins = 5; // how often to 68->67
 double physicsParams.nominalDAQFrequency = 40.; // MHz
 
 // charged pion and kaon
-vector<int> physicsParams.freeLifePDGId = { 211, 312 };
+vector<int> physicsParams.freeLifePDGId   = {  211   , 312   ,  13    , 2212};
```

Comment:
What is the use case for setting the muon lifetime here?  Is the point to update the value?  It seems odd that only these few particles livetimes are set.
We need to use a single standard source for particle properties, please talk to the simulation coordinator to figure out what that is.